planb/create_cluster.py

```diff
@@ -889,6 +890,10 @@ def extend_cluster(options: dict):
         )
         launch_seed_nodes(options)
 
+        if options['allocate_tokens_for_keyspace']:
+            env['AUTO_BOOTSTRAP'] = 'true'
```

Comment:
Rather than setting this value twice and overriding I would propose to have function for its final value.